test/FunctionsV2/EntityMethodDiscoveryTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.Azure.WebJobs;
+using Moq;
+using Xunit;
+
+namespace WebJobs.Extensions.DurableTask.Tests.V2
+{
+    public class EntityMethodDiscoveryTests
+    {
+        private interface ICounter
+        {
+            void Add(int count);
+        }
+
+        [Fact]
```

Comment:
I wasn't sure what (if any) `Trait` to put on the tests. Let me know if it's needed and which one.

Reply:
Added to all tests